images/ubuntu/Ubuntu2204-Readme.md

```diff
@@ -87,19 +87,19 @@ to accomplish this.
 - jq 1.6
 - Kind 0.20.0
 - Kubectl 1.28.4
-- Kustomize 5.2.1
+- Kustomize 5.3.0
 - Leiningen 2.10.0
 - MediaInfo 21.09
 - Mercurial 6.1.1
 - Minikube 1.32.0
 - n 9.2.0
 - Newman 6.0.0
-- nvm 0.39.6
+- nvm 0.39.7
 - OpenSSL 3.0.2-0ubuntu1.12
-- Packer v1.10.0
+- Packer Packer v1.10.0
```

Comment:
@Alexey-Ayupov @erik-bershel  could we fix the function call in sw report?

Reply:
Will do